code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -124,23 +127,30 @@
 
 	repeatable = TRUE
 
+/datum/dynamic_ruleset/latejoin/ninja/ready(var/forced = 0)
```

Comment:
I'm not sure this is the best place for the prompt to happen, since ready gets called on all rulesets before one of them is chosen. Maybe `execute` is the right proc instead. @ShiftyRail

Reply:
Functionally, there'd be no difference, since all the rulesets get drafted and thus the timer gets reset anyhow. It also only tries to run it off that one latejoiner. If anything, I should move it *up* the chain if I wanted things to change.

Reply:
What I'm saying is that if you click yes but then another ruleset gets chosen, it's an "uhhh what the fuck is going on?" kinda situation. Like, click yes on the prompt to become ninja, become wizard instead.

Reply:
hm, i see your point